src/code/z_demo.c

```diff
@@ -235,96 +247,118 @@ void Cutscene_Command_Misc(PlayState* play, CutsceneContext* csCtx, CsCmdBase* c
             break;
-        case 0x11:
+
+        case CS_MISC_SUNSSONG_START:
             gSaveContext.sunsSongState = SUNSSONG_START;
             break;
-        case 0x12:
+
+        case CS_MISC_FREEZE_TIME:
             if (!gSaveContext.save.isNight) {
                 gSaveContext.save.time = ((void)0, gSaveContext.save.time) - (u16)R_TIME_SPEED;
             } else {
                 gSaveContext.save.time = ((void)0, gSaveContext.save.time) - (u16)(2 * R_TIME_SPEED);
             }
             break;
-        case 0x13:
-            AudioOcarina_PlayLongScarecrowAfterCredits();
-            csCtx->frames = cmd->startFrame - 1;
+
+        case CS_MISC_LONG_SCARECROW_SONG:
+            AudioOcarina_PlayLongScarecrowSong();
+            csCtx->curFrame = cmd->startFrame - 1; // the cutscene runs forever
             break;
-        case 0x14:
-            EnvFlags_Set(play, 3);
+
+        case CS_MISC_SET_CSFLAG_3:
+            CutsceneFlags_Set(play, 3);
             break;
-        case 0x15:
-            EnvFlags_Set(play, 4);
+
+        case CS_MISC_SET_CSFLAG_4:
+            CutsceneFlags_Set(play, 4);
             break;
-        case 0x16:
+
+        case CS_MISC_PLAYER_FORM_DEKU:
             gSaveContext.save.playerForm = PLAYER_FORM_DEKU;
             break;
-        case 0x17:
+
+        case CS_MISC_ENABLE_PLAYER_REFLECTION:
             player->stateFlags2 |= PLAYER_STATE2_4000000;
             break;
-        case 0x18:
+
+        case CS_MISC_DISABLE_PLAYER_REFLECTION:
             player->stateFlags2 &= ~PLAYER_STATE2_4000000;
             break;
-        case 0x19:
+
+        case CS_MISC_PLAYER_FORM_HUMAN:
             sCutsceneStoredPlayerForm = gSaveContext.save.playerForm;
             gSaveContext.save.playerForm = PLAYER_FORM_HUMAN;
             gSaveContext.save.equippedMask = PLAYER_MASK_NONE;
             break;
-        case 0x1A:
+
+        case CS_MISC_STRONG_QUAKE_START:
             func_8019F128(NA_SE_EV_EARTHQUAKE_LAST2 - SFX_FLAG);
-            if (isStartFrame) {
+            if (isFirstFrame) {
                 sCutsceneQuakeIndex = Quake_Add(GET_ACTIVE_CAM(play), QUAKE_TYPE_6);
                 Quake_SetSpeed(sCutsceneQuakeIndex, 30000);
                 Quake_SetQuakeValues(sCutsceneQuakeIndex, 20, 10, 0, 0);
                 Quake_SetCountdown(sCutsceneQuakeIndex, 800);
             }
             break;
-        case 0x1B:
-            if (isStartFrame) {
+
+        case CS_MISC_DEST_MOON_CRASH_FIRE_WALL:
+            if (isFirstFrame) {
                 play->nextEntrance = ENTRANCE(CUTSCENE, 0);
                 gSaveContext.nextCutsceneIndex = 0xFFF8;
                 play->transitionTrigger = TRANS_TRIGGER_START;
                 play->transitionType = TRANS_TYPE_FADE_WHITE;
             }
             break;
-        case 0x1C:
-            if (isStartFrame) {
+
+        case CS_MISC_SKYBOX_MOON_CRASH:
+            if (isFirstFrame) {
+                // skyboxConfig
```

Comment:
`CS_MISC_MOON_CRASH_SKYBOX` maybe, and has symmetry with `_MOON_CRASH_FIRE_WALL` above (tho the symmetry isnt that important, I think it just reads better)